core/domain/activity_domain.py

```diff
@@ -56,7 +56,12 @@ def validate(self):
             raise Exception('Invalid activity type: %s' % self.type)
 
     def to_dict(self):
-        """Returns a dict representing this ActivityReference domain object."""
+        """Returns a dict representing this ActivityReference domain object.
+
+        Returns:
+            A dict, mapping activity_type and activity_id of a ActivityReference
```

Comment:
There's no reference to activity_type and activity_id here. Maybe you mean just 'type' and 'id'.